### PR TITLE
Further improve the language of the first two pages

### PR DESCRIPTION
**Pull request by jayp**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -3,57 +3,59 @@ title: Introduction
...
+
 # Introduction
 
 - [Jay A. Patel](https://twitter.com/jayisms) <<hello@datadriventeam.org>>
 - [Nilkanth Patel](https://twitter.com/nilkanthjp) <<hello@datadriventeam.org>>
 
 ## Preface
 
-Over the past few months, Neel and I have spoken with 50+ companies to figure out how they "make something people want." This guide summarizes our learnings from **processes run at customer-oriented companies like Brex, Notion, Gem, Tandem, and more**.
+Over the past few months, Nilkanth (Neel) and I have spoken with 50+ companies to understand how companies are iterating on "make something people want." We are chronicling the many of these conversations in our blog: [Userstand— how innovative companies understand their users](https://www.heraldhq.com/userstand/).
```
Comment (nilkanthjp) [inline content/_index.md:17]: Don't really know what "iterating on "make something people want" means. Also, don't think you need the userstand tagline as you're basically saying that's what the blog is about in the previous sentence.

File: content/_index.md [hunk 2]
```diff
@@ -3,57 +3,59 @@ title: Introduction
...
 
 {{< /columns >}}
 
-The key difference between the two is that **product analytics is objective** where as **user feedback is subjective**. Ultimately, answering questions quantitatively by looking at product analytics is a backbone of a data-driven team. However, while product analytics can answer a lot of what **users do** and **don't do**, they can't answer the **why** or **why nots** behind those user actions. The best way to understand the motivations behind those actions is by asking the user -- by encouraging and nurturing user feedback. As we observed at many of these customer-oriented companies, when user feedback is an integral part of their engineering processes, it becomes the flywheel that empowers your team to pinpoint and solve the most important customer painpoints.
+The key difference between the two is that **product analytics is objective** where as **user feedback is subjective**. Ultimately, answering questions quantitatively by looking at product analytics is a backbone of a data-driven team. However, while product analytics can answer a lot of what **users do** and **don't do**, they can't answer the **why** or **why nots** behind those user actions.
```
Comment (nilkanthjp) [inline content/_index.md:41]: where as -> whereas

Comment (nilkanthjp) [reply content/_index.md:41]: a backbone -> the backbone

Comment (nilkanthjp) [reply content/_index.md:41]: they can't answer the **why** or **why nots** behind those user actions -> they can't answer the **why** or **why not**

File: content/docs/200-process.md.bak [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (nilkanthjp) [inline content/docs/200-process.md.bak:1]: What is a .bak file?

File: content/_index.md (hunk 2 above)
Comment (jayp) [reply content/_index.md:41]: Just singularize not? Or also drop the rest of the sentence? Assuming the former.

File: content/docs/200-process.md.bak (hunk 3 above)
Comment (jayp) [reply content/docs/200-process.md.bak:1]: misake

File: content/_index.md (hunk 2 above)
Comment (nilkanthjp) [reply content/_index.md]: I meant the latter actually. I think the phrasing still needs work (why or why nots is a kind of weird phrase), but can defer.

**Pull request merged**